wordvecspace/server.py

```diff
@@ -160,27 +165,28 @@ def get_distances(self, row_words_or_indices: Union[str, int, tuple, list], col_
         get_distances(word, words)
         get_distances(words_x, words_y)
 
-        get_distances("for", ["to", "for", "india"] => [[  1.4990e-01], [ -1.1921e-07], [  1.3855e+00]]
-        get_distances("for", ["to", "for", "inidia"]) => [[  1.4990e-01], [ -1.1921e-07], [  1.0000e+00]]
-        get_distances(["india", "for"], ["to", "for", "usa"]) => [[  1.1830e+00,   1.3855e+00,   4.8380e-01], [  1.4990e-01,  -1.1921e-07,   1.4975e+00]]
-        get_distances(["india", "usa"]) => [[ 1.4903,  0.4202,  0.269 , ...,  1.2041,  1.3539,  0.6154], [ 1.8084,  0.9541,  1.1678, ...,  0.5963,  1.0458,  1.1608]]
-        get_distances(["andhra"]) => [[ 1.3432,  0.5781,  0.2306, ...,  1.0937,  1.1369,  0.4284]]
-        get_distances(["andhra"], metric='euclidean') => [[ 1.601   1.108   0.7739 ...,  1.4103  1.5646  1.1079]]
+        get_distances("for", ["to", "for", "india"] => [[  2.7428e-01,   5.9605e-08,   1.1567e+00]]
+        get_distances("for", ["to", "for", "inidia"]) => [[  2.7428e-01,   5.9605e-08,   1.0000e+00]]
+        get_distances(["india", "for"], ["to", "for", "usa"]) => [[[  1.1445e+00   1.1567e+00   3.7698e-01], [  2.7428e-01   5.9605e-08   1.6128e+00]]
+        get_distances(["india", "usa"]) => [[ 1.5464  0.4876  0.3017 ...,  1.2492  1.2451  0.8925], [ 1.0436  0.9995  1.0913 ...,  0.6996  0.8014  1.1608]]
+        get_distances(["andhra"]) => [[ 1.5418  0.7153  0.277  ...,  1.1657  1.0774  0.7036]]
+        get_distances(["andhra"], metric='euclidean') => [[ 1.756   1.1961  0.7443 ...,  1.5269  1.4679  1.1862]]
         '''
-        if self._type == 'mem':
+        if self._type == 'mem' or 'disk':
             return self.wv.get_distances(row_words_or_indices, col_words_or_indices=col_words_or_indices, metric=metric, raise_exc=raise_exc).tolist()
 
         return self.wv.get_distances(row_words_or_indices, col_words_or_indices=col_words_or_indices, raise_exc=raise_exc).tolist()
 
     def get_nearest(self, words_or_indices: Union[str, int, list, tuple], k: int=512, metric: Union[str, None]=None, combination: bool=False, raise_exc: bool=False) -> list:
```

Comment:
@prasannababuAddagiri here we are not following the pep-8 standard's can you please check it once.